docker/docker-compose.1804.42.yaml

```diff
@@ -15,9 +15,21 @@ services:
 
   integration-tests:
     image: swift-nio:18.04-4.2
+    environment:
+      - MAX_ALLOCS_ALLOWED_1000_reqs_1_conn=36750
```

Comment:
CC @eeckstein, @gottesmm, @airspeedswift, @milseman (random list of people CC'd that I know helped reducing these limits) the NIO team say thanks for this: Check this out:

| test | allocations in 4.1 | allocations in 4.2 |
| ----| ----| ---- |
| 1000 HTTP/1.1 requests over 1 connection | 47 allocations per request/response pair | 36 allocs for req/res pair |
| 1 HTTP/1.1 request per connection with 1000 connections | 719 allocs per req/res pair | 698 allocs |

Also CC @tanner0101 / @LoganWright and @ianpartridge / @pushkarnk who'll be happy to know we're down 20% of the allocations (from 47 to 36) to for a HTTP/1.1 request response pair, that's pretty major for 0 NIO code change. And wait for Swift 5.0 💛

Reply:
wow... thats really a lot!

Reply:
I know right!

Reply:
🍻 for the Swift team!!

Reply:
🎉🎉🎉